### ISSUE TITLE
 Use /usr/bin/python3 shebang

### ISSUE THREAD

**Issue by edigiacomo** [ARPA-SIMC/dballe#131]
From https://fedoraproject.org/wiki/Packaging:Python#Multiple_Python_Runtimes:

> Packages in Fedora MUST NOT use /usr/bin/python. Instead packages for Python 3 MUST use /usr/bin/python3 (even if upstream supports both Python 2 and 3). As a result of that /usr/bin/python (as well as /usr/bin/env python and similar) MUST NOT be used in shebang lines or as a dependency of a package. As of Fedora 30, all uses of unversioned python executables in shebang lines will fail the build

This issue should be closed after #127.

**Comment by spanezz** [comment]
In the current branch `issue122` there should be no remaining python shebangs. This can be closed once it will be merged into master.

**Comment by spanezz** [comment]
Closing after merge to master
